enterprise_gateway/services/processproxies/processproxy.py

```diff
@@ -361,8 +361,9 @@ def _enforce_authorization(self, **kw):
             env_dict['KERNEL_USERNAME'] = kernel_username
 
         # Now perform authorization checks
-        if kernel_username in self.unauthorized_users:
-            self._raise_authorization_error(kernel_username, "not authorized")
+        if self.unauthorized_users.__len__() > 0:
```

Comment:
I don't think this is necessary.  This was only used for the authorized check because its a `not in` expression, which would be true if the list is empty.